File: fogl/js/PointerPeer.js
//
// Copyright (c) 2011, chunquedong
// Licensed under the Academic Free License version 3.0
//
// History:
//   2011-5-29  Jed Young  Creation
//


fan.fogl.PointerPeer = fan.sys.Obj.$extend(fan.sys.Obj);
fan.fogl.PointerPeer.prototype.$ctor = function(self) {}
fan.fogl.PointerPeer.prototype.m_val = null;
fan.fogl.PointerPeer.prototype.val = function(self){ return "" + this.m_val; }
fan.fogl.PointerPeer.prototype.val$ = function(self, i) { this.m_val = i; }

File: fogl/java/PointerPeer.java
//
// Copyright (c) 2011, chunquedong
// Licensed under the Academic Free License version 3.0
//
// History:
//   2011-5-31  Jed Young  Creation
//

package fan.fogl;

import fan.sys.*;

class PointerPeer
{
  long pointer;

  public static PointerPeer make(Pointer self)
  {
    return new PointerPeer();
  }

  long val(Pointer self)
  {
    return pointer;
  }

  void val(Pointer self, long i)
  {
    pointer = i;
  }
}